### ISSUE TITLE
Workflows can't be re-ordered with the keyboard

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/Panoptes-Front-End#5002]
## Expected behavior
When I select 'Reorder view' from the workflows page in the project builder, I should be able to select a workflow then move it within the list using the up and down cursor keys.

## Current behavior
_Please include any error messages from the browser console and/or screenshots_
Drag and drop only works with a mouse or touchscreen.

## Additional information
This is possibly a lack of keyboard support in [zooniverse/drag-reorderable](https://github.com/zooniverse/drag-reorderable)

- **Operating system:** N/A
- **Browser:** N/A
